### DISCUSSION TITLE
Add PHP "after" getListQuery Method

### DISCUSSION THREAD

**Discussion by saman222** [joomengine/Joomla-Component-Builder#687] [Ideas]
hi
it seems very good if i can show anydata in my admin view column. if it is possible we can use joined list after create joined table we have a more useful section for php. what is your idea?

**Comment by Llewellynvdm** [comment]
You can add joints at that moment as well as the database object `$query` is already initialized, and the order of adding the queries does not matter.

![image](https://user-images.githubusercontent.com/5607939/110216645-deae9e00-7eb8-11eb-9fc5-ee9acdac56c3.png)
